aiinfra-cluster/modules/aiinfra-compute/main.tf

```diff
@@ -22,14 +22,10 @@ locals {
 
   resource_prefix = var.name_prefix != null ? var.name_prefix : var.deployment_name
 
-  enable_gvnic  = var.bandwidth_tier != "not_enabled"
-  enable_tier_1 = var.bandwidth_tier == "tier_1_enabled"
+  is_gvnic_supported = (var.instance_image.name != null && length(regexall("debian-11", var.instance_image.name)) > 0) || (var.instance_image.family != null && length(regexall("debian-11", var.instance_image.family)) > 0) || (var.instance_image.name != null && length(regexall("ubuntu", var.instance_image.name)) > 0) || (var.instance_image.family != null && length(regexall("ubuntu", var.instance_image.family)) > 0) || (var.instance_image.name != null && length(regexall("gvnic", var.instance_image.name)) > 0) || (var.instance_image.family != null && length(regexall("gvnic", var.instance_image.family)) > 0)
```

Comment:
nitpick: does terraform allow huge conditionals to be broken up with some kind of backslash action?

Reply:
yup. Found a way to do that.